src/Security/Authentication/Cookies/src/CookieAuthenticationOptions.cs

```diff
@@ -128,8 +128,7 @@ public CookieBuilder Cookie
     /// even if it is passed to the server after the browser should have purged it.
     /// </para>
     /// <para>
-    /// This is separate from the value of <see cref="CookieOptions.Expires"/>, which specifies
-    /// how long the browser will keep the cookie.
+    /// This property should be used instead of <see cref="CookieOptions.Expires"/>, which is ignored.
```

Comment:
nit: Can this be `Cookie.Expires` since that is the code you'd write to access the ignored property? Or `CookieBuilder.Expires` since that's the type of the `Cookie` property on `CookieAuthenticationOptions`.